docs/10-clusters.md

```diff
@@ -27,6 +27,32 @@ NOTE: The cluster id (--id) can only contain alphanumeric characters including h
 #### Low priority nodes
 You can create your cluster with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using **--size-low-pri** instead of **--size**. Note that these are great for experimental use, but can be taken away at any time. We recommend against this option when doing long running jobs or for critical workloads.
 
+#### Configuration
+Set your cluster's configuration in a single location using the config/cluster.yaml file. A template with all available configuration settings is provided at config/cluster.yaml.template.
+
+For example, with the following configuration:
```

Comment:
@timotheeguerin - we chatted about having a command that will create the config folder (.thunderbolt) - are we still doing that for this PR? 

What is the command? azb spark cluster generate-config

BTW
I'm thinking that we should call it .thunderbolt-cluster now because eventually we'll have a .thunderbolt-job right?
So we'll potentially also need a: azb spark job generate-config